src/app/convert-string-to-dots.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { ConvertStringToDotsPipe } from './convert-string-to-dots.pipe';
```

Comment:
File can be moved to a utils directory to keep the application structure clean

Reply:
Create the pipe with the following command `ng generate pipe pipeName` probably located on the default location for now. Do agree that it's a good idea to keep the structure clean. Will also move the vacancy class and IVacancies interface to separate directories.